chart/templates/deployment.yaml

```diff
@@ -83,6 +83,10 @@ spec:
               fieldPath: metadata.uid
         - name: KUBERNETES_CLUSTER_DOMAIN
           value: {{ .Values.controller.kubernetesClusterDomain }}
+        {{- range .Values.controller.manager.extraEnvs }}
+        - name: {{ .name }}
+          value: {{ .value | quote }}
```

Comment:
Could this result in double quoting the value e.g.: `.value = "foo"`

Reply:
No, `quote` doesn't double quote already quoted strings. It's best practice to always pipe inputs expected to be strings or env values to `quote`, [according to the Helm documentation](https://helm.sh/docs/howto/charts_tips_and_tricks/#quote-strings-dont-quote-integers).